src/v2/view-builder/views/webauthn/ChallengeWebauthnAuthenticatorView.js

```diff
@@ -0,0 +1,142 @@
+import { loc, _, createButton } from 'okta';
+import BaseForm from '../../internals/BaseForm';
+import BaseFooter from '../../internals/BaseFooter';
+import BaseFactorView from '../shared/BaseFactorView';
+import CryptoUtil from '../../../../util/CryptoUtil';
+import webauthn from '../../../../util/webauthn';
+import ChallengeWebauthnInfoView from './ChallengeWebauthnInfoView';
+
+const Body = BaseForm.extend({
+
+  title () {
+    return loc('oie.verify.webauth.title', 'login');
+  },
+
+  className: 'oie-verify-webauthn',
+
+  getUISchema () {
+    const schema = [];
+    // Returning custom array so no input fields are displayed for webauthn
+    if (webauthn.isNewApiAvailable()) {
```

Comment:
I was wondering do you need to display `webauthn not supported` like in enroll case.
And looks like you trigger `error` instead. Curious why you cant do similar as Enroll (push callout into ui schema)?